Common/src/CConfig.cpp

```diff
@@ -6008,6 +6009,35 @@ void CConfig::SetMarkers(SU2_COMPONENT val_software) {
     Marker_CfgFile_MixingPlaneInterface[iMarker_CfgFile] = indexMarker;
   }
 
+  /*--- Once we have identified the MixingPlane and Turbomachinery markers
+   *    we next need to determine what type of interface between zones is
+   *    used. It is convenient to do this here as it tidies up the interface
+   *    preproccesing in CDriver ---*/
+  nTurboInterfaces = (nMarker_Turbomachinery -  1)*2; //Two markers per zone
```

Comment:
Could you explain the logic behind calculating the sizes? Why the `- 1`? At first glance, it looks like this could be conflicting with the `nMarker_Turbomachinery*2-1` further below, but I havn't checked this in detail and it might be correct.

Reply:
nMarker_Turbomachinery is equal to half the number of markers you specify in MARKER_TURBOMACHINERY, this is because of line 1653 in option_structure.inl

`unsigned long nVals = totalVals / mod_num;
    this->size = nVals;`

I am not sure if this is a mistake, however this is used frequently in CPhysicalGeometry for defining the TurboVertex structure and it was simpler to only modify the value here, than have to change it in many multiple more places if it is a mistake. We take off 1 and multiply by 2 as we are only interested in the markers specified in MARKER_TURBOMACHINERY which are interfaces between zones, i.e. remove the inlet and outlet. I have added a comment to make this clearer in the code.